diff --git a/lib/logging.js b/lib/logging.js
index 1ca25093178dcec6d38fa3c0a46603b3522d54dc..3d6202321527a06f5c03a49490591b7a2e20543e 100644
--- a/lib/logging.js
+++ b/lib/logging.js
@@ -24,10 +24,7 @@ __export(logging_exports, {
 module.exports = __toCommonJS(logging_exports);
 var import_pino = require("pino");
 const logger = (0, import_pino.pino)({
-  level: "error",
-  transport: {
-    target: "pino-pretty"
-  }
+  level: "error"
 });
 function setLogLevel(level) {
   logger.level = level;
diff --git a/lib/logging.mjs b/lib/logging.mjs
index 875d4d57620eeabf180ac6d5a795002e0a7955fb..6e913a3398909080d717ce51a303ded1515cc751 100644
--- a/lib/logging.mjs
+++ b/lib/logging.mjs
@@ -1,10 +1,7 @@
 import "./chunk-4ZNVX5QM.mjs";
 import { pino } from "pino";
 const logger = pino({
-  level: "error",
-  transport: {
-    target: "pino-pretty"
-  }
+  level: "error"
 });
 function setLogLevel(level) {
   logger.level = level;
diff --git a/src/logging.ts b/src/logging.ts
index cb6bd312af369355bed139bd394c32b118ad0f7c..9626e5c27cc5487efe750f12669071b47403a905 100644
--- a/src/logging.ts
+++ b/src/logging.ts
@@ -18,9 +18,6 @@ import { pino } from 'pino';
 
 const logger = pino({
   level: 'error',
-  transport: {
-    target: 'pino-pretty',
-  },
 });
 
 /**
