diff --git a/lib/index.js b/lib/index.js
index fcf8aa85e8e7b99a0d9d3eef1b26504dc9519f72..492421285be3beb2bc12f2502bee76eed0fde164 100644
--- a/lib/index.js
+++ b/lib/index.js
@@ -56,7 +56,9 @@ var import_google_auth_library = require("google-auth-library");
 const vertexAI = (0, import_config.genkitPlugin)(
   "vertex-ai",
   (options) => __async(void 0, null, function* () {
-    const authClient = new import_google_auth_library.GoogleAuth();
+    const authClient = new import_google_auth_library.GoogleAuth({
+      scopes: 'https://www.googleapis.com/auth/cloud-platform'
+    });
     const project = options.projectId || (0, import_common.getProjectId)();
     const location = options.location || (0, import_common.getLocation)();
     const confError = (parameter, envVariableName) => {
diff --git a/lib/index.mjs b/lib/index.mjs
index 622f87b6391ebcf4feac5ed038b0e83f3aa72a1f..f2b1ed5b3fa313d43e2405e56bfea7021028ca24 100644
--- a/lib/index.mjs
+++ b/lib/index.mjs
@@ -16,7 +16,9 @@ import { GoogleAuth } from "google-auth-library";
 const vertexAI = genkitPlugin(
   "vertex-ai",
   (options) => __async(void 0, null, function* () {
-    const authClient = new GoogleAuth();
+    const authClient = new GoogleAuth({
+      scopes: 'https://www.googleapis.com/auth/cloud-platform'
+    });
     const project = options.projectId || getProjectId();
     const location = options.location || getLocation();
     const confError = (parameter, envVariableName) => {
diff --git a/src/index.ts b/src/index.ts
index 79f862df0217b1f7575e3e5681cb281fe98ac66a..382cf1b5f76b7a925ae7806ba48e45060fb0f66d 100644
--- a/src/index.ts
+++ b/src/index.ts
@@ -37,7 +37,9 @@ export interface PluginOptions {
 export const vertexAI: Plugin<[PluginOptions]> = genkitPlugin(
   'vertex-ai',
   async (options: PluginOptions) => {
-    const authClient = new GoogleAuth();
+    const authClient = new GoogleAuth({
+      scopes: 'https://www.googleapis.com/auth/cloud-platform',
+    });
     const project = options.projectId || getProjectId();
     const location = options.location || getLocation();
 
